src/pages/public_registration/action_accordions/FindRegistration.tsx

```diff
@@ -0,0 +1,122 @@
+import { Registration } from '../../../types/registration.types';
+import { Autocomplete, Box, IconButton, TextField, Typography } from '@mui/material';
+import { dataTestId } from '../../../utils/dataTestIds';
+import SearchIcon from '@mui/icons-material/Search';
+import { useQuery } from '@tanstack/react-query';
+import { fetchResults } from '../../../api/searchApi';
+import { useState } from 'react';
+import { useDebounce } from '../../../utils/hooks/useDebounce';
+import { useTranslation } from 'react-i18next';
+import { ErrorBoundary } from '../../../components/ErrorBoundary';
+import { SearchListItem } from '../../../components/styled/Wrappers';
+import { RegistrationListItemContent } from '../../../components/RegistrationList';
+
+function isTitle(query: string) {
+  return !isDoi(query) && !isHandle(query);
+}
+
+function isDoi(query: string) {
+  return query.includes('https://doi.org/');
+}
+
+function isHandle(query: string) {
+  return query.includes('https://hdl.handle.net/');
+}
+
+interface FindRegistrationProps {
+  setSelectedRegistration: (registration: Registration | null) => void;
+  selectedRegistration: Registration | null;
+  filteredRegistrationIdentifier: string;
+}
+export const FindRegistration = ({
+  setSelectedRegistration,
+  selectedRegistration,
+  filteredRegistrationIdentifier,
+}: FindRegistrationProps) => {
+  const { t } = useTranslation();
+  const [searchBeforeDebounce, setSearchBeforeDebounce] = useState('');
+  const debouncedSearch = useDebounce(searchBeforeDebounce);
+
+  const fetchQuery = {
+    title: isTitle(debouncedSearch) ? debouncedSearch : null,
+    doi: isDoi(debouncedSearch) ? debouncedSearch : null,
+    query: isHandle(debouncedSearch) ? debouncedSearch : null,
+  };
+
+  const duplicateRegistrationSearch = useQuery({
+    enabled: debouncedSearch.length > 0,
+    queryKey: ['duplicateRegistration', fetchQuery],
+    queryFn: () => fetchResults(fetchQuery),
+    meta: { errorMessage: t('feedback.error.get_registrations') },
+  });
+
+  const handleSearchAutoCompleteChange = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    setSearchBeforeDebounce(event.target.value);
+  };
+
+  const searchResults = duplicateRegistrationSearch.data?.hits ?? [];
+
+  const searchResultNotContainingToBeDeleted = searchResults.filter(
+    (possibleDuplicate) => possibleDuplicate.identifier !== filteredRegistrationIdentifier
+  );
+
+  const defaultProps = {
```

Comment:
noen grunn til å ta dette utenfor?
Fremstår litt mystisk nå med hvorfor `options` er en "defaultProp", men ikke `freeSolo` og alle andre som ser like "default" ut om du skjønner 🤔 En fordel om den er inni er at MUI sikrer at alle types er riktig, men det kan sikres ved å gjøre det sånn også da, _om_ man også setter riktig type for objektet med `const defaultProps: AutocompleteProps<...>`, men da må man i praksis bare vedlikeholde typen selv, samtidig som MUI automatisk også vil gjøre det

Reply:
Satt og knotet med dette og glemte å rydde opp. Fikset.